src/cljs/webchange/ui_framework/layout/navigation/views.cljs

```diff
@@ -1,10 +1,31 @@
 (ns webchange.ui-framework.layout.navigation.views
   (:require
     [re-frame.core :as re-frame]
-    [webchange.routes :refer [location redirect-to]]
     [webchange.state.core :as state]
-    [webchange.ui-framework.components.index :refer [button]]
-    [webchange.ui-framework.components.utils :refer [get-class-name]]))
+    [webchange.auth.subs :as as]
+    [webchange.routes :refer [location redirect-to]]
+    [webchange.ui-framework.components.utils :refer [get-class-name]]
+    [webchange.ui-framework.layout.avatar.views :refer [avatar]]
+    [webchange.ui-framework.layout.logo.views :refer [logo]]
+    [webchange.ui-framework.layout.navigation.svg :as user-icn]))
+
+(def menu-items
+  [{:title         "Dashboard"
+    :location-name :welcome-screen}
```

Comment:
I suppose we will need unique icon for each menu item, so lets store icons names here.